SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -125,4 +134,6 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	Worker_RequestId NextRequestId = 0;
 
 	ESpatialConnectionType ConnectionType = ESpatialConnectionType::Receptionist;
+    
+    LoginTokenCb    LoginTokenCb_ = NULL;
```

Comment:
Don't use underscores for member fields.

Reply:
Done